### PR TITLE
Page object pattern feature added

### PR DESCRIPTION
**Pull request by avoloshchuk567**

### CODE REVIEW

File: src/test/java/pageObjectClasses/CreatedUserAccount.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    @FindBy(css = "table[border='1'] > tbody > tr:first-child > td:last-child > b ")
+    private WebElement userAccountEmail;
+    @FindBy(css = "table[border='1'] > tbody > tr:last-child > td:last-child")
+    private WebElement userAccountPassword;
+    @FindBy(css = "span[class='confirmation_message']")
+    private WebElement confirmationMessage;
+    @FindBy(css = "a[href*='./main.py']")
+    private WebElement homePage;
+
+    public String getUserAccountEmail() {
```
Comment (uvidit) [inline src/test/java/pageObjectClasses/CreatedUserAccount.java:26]: All methods in your page object classes could be marked with @Step - it'll add the flow visibility to the allure report

File: src/test/java/pageObjectTests/MainPageTest.java [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        driver.quit();
+    }
+
+    @Test(description = "Test of Email text on Main page")
+    @Description("Test description: check that email text is correct")
+    public void checkEmailTextIsCorrectTest() {
+        mainPage = loadMainPage();
+        String emailText = mainPage.getEmailFieldText();
+        assertThat(emailText, equalTo("Email"));
+    }
```
Comment (uvidit) [inline src/test/java/pageObjectTests/MainPageTest.java:58]: >         mainPage = loadMainPage();
>         String emailText = mainPage.getEmailFieldText();
>         assertThat(emailText, equalTo("Email"));
Variables 'mainPage' and 'emailText' used only once , so test code could be more clear: 

```
assertThat(loadMainPage().getEmailFieldText(), equalTo("Email"));
``` 
or at least :
```
String emailText = loadMainPage().getEmailFieldText();
assertThat(emailText, equalTo("Email"));
```

File: src/test/java/pageObjectTests/MainPageTest.java [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+    @Test(description = "Test of Password text on Main page")
+    @Description("Test description: check that password text is correct")
+    public void checkPasswordTextIsCorrectTest() {
+        mainPage = loadMainPage();
+        String passwordText = mainPage.getPasswordFieldText();
+        assertThat(passwordText, equalTo("Password"));
+    }
```
Comment (uvidit) [inline src/test/java/pageObjectTests/MainPageTest.java:64]: >  mainPage = loadMainPage();
>  String passwordText = mainPage.getPasswordFieldText();
>  assertThat(passwordText, equalTo("Password"));

the same here and then below - could be simplified and be more clear

**Review by uvidit — CHANGES_REQUESTED** [review]

File: src/test/java/pageObjectClasses/CreatedUserAccount.java (hunk 1 above)
Comment (avoloshchuk567) [reply src/test/java/pageObjectClasses/CreatedUserAccount.java:26]: @Step annotations in PageObject classes were added in 650df7fc4fecbff7c663b7d51b9574ad6cdd033f.

File: src/test/java/pageObjectTests/MainPageTest.java (hunk 2 above)
Comment (avoloshchuk567) [reply src/test/java/pageObjectTests/MainPageTest.java:58]: Changed to requested in 650df7fc4fecbff7c663b7d51b9574ad6cdd033f.

File: src/test/java/pageObjectTests/MainPageTest.java (hunk 3 above)
Comment (avoloshchuk567) [reply src/test/java/pageObjectTests/MainPageTest.java:64]: Changed to requested in 650df7fc4fecbff7c663b7d51b9574ad6cdd033f.

**Pull request merged**